### PR TITLE
Refactor/simplify initializer

### PR DESCRIPTION
**Pull request by Rafe**
Simplify initializer logic

### CODE REVIEW

File: app/assets/javascripts/i18n.js [hunk 1]
```diff
@@ -55,6 +55,11 @@
     return type === 'function' || type === 'object' && !!obj;
   };
 
+  // Check if value is different than undefined and null;
+  var isSet = function(value) {
+    return value !== undefined && value !== null;
+  }
```
Comment (PikachuEXE) [inline app/assets/javascripts/i18n.js]: missing semicolon

File: app/assets/javascripts/i18n.js [hunk 2]
```diff
@@ -55,6 +55,11 @@
     return type === 'function' || type === 'object' && !!obj;
   };
 
+  // Check if value is different than undefined and null;
+  var isSet = function(value) {
+    return value !== undefined && value !== null;
```
Comment (PikachuEXE) [inline app/assets/javascripts/i18n.js]: I think the common check should be `typeof value !== "undefined" && value !== null`
This avoids accident when `undefined` assigned to something else

Comment (Rafe) [inline app/assets/javascripts/i18n.js]: fixed

**Comment by PikachuEXE** [comment]
Thanks!
Will also look at other PR soon

**Pull request merged**